### ISSUE TITLE
Add a shortcut for show/hide the dock.

### ISSUE THREAD

**Issue by morisrc** [micheleg/dash-to-dock#771]
It would be really handy to have a shortcut to show / hide the dock, without having to go through the settings. It's possible?

**Comment by morisrc** [comment]
Exactly. I tend to keep the dock hidden because it distracts me, but in some cases it is convenient to have it always visible. I think a keyboard shortcut is the easiest and fastest way to avoid gnome-tweak.

**Comment by aspizu** [comment]
Could you reconsider this? Something like Super + S, similar to Zen/Arc browser sidebars
